pkg/asset/agent/mirror/registriesconf.go

```diff
@@ -162,9 +162,8 @@ func (i *RegistriesConf) Generate(dependencies asset.Parents) error {
 	i.Config = registries
 	i.setMirrorConfig(i.Config)
 
-	releaseImagePath := strings.Split(releaseImage.PullSpec, ":")[0]
-	if found := i.validateReleaseImageIsSameInRegistriesConf(releaseImagePath); !found {
-		logrus.Warnf(fmt.Sprintf("The ImageContentSources configuration in install-config.yaml should have at-least one source field matching the releaseImage value %s", releaseImagePath))
+	if !i.validateReleaseImageIsSameInRegistriesConf(releaseImage.PullSpec) {
```

Comment:
nit: removing 'validate' from the function name would make the meaning of the return value more clear.

Reply:
Good point. Changed.